source/export-to-language.txt

```diff
@@ -0,0 +1,44 @@
+.. _vsce-export-to-language:
+
+======================================
+Export a Query or Pipeline to Language
+======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can export query documents and aggregation pipelines from a
+:ref:`playground <vsce-playgrounds>` to a programming language. You can
```

Comment:
I'd like to stick with export generally because that is the language used in the UI. I think calling it "translate" might confuse users since they can map what they see in the docs 1:1 to what's in the extension UI.

I will try to work "translate" into this opening sentence.

Reply:
Done